sparkmagic/sparkmagic/magics/remotesparkmagics.py

```diff
@@ -52,6 +53,8 @@ def manage_spark(self, line, local_ns=None):
                                                                         "from the server for SQL queries")
     @argument("-r", "--samplefraction", type=float, default=None, help="Sample fraction for sampling from SQL queries")
     @argument("-u", "--url", type=str, default=None, help="URL for Livy endpoint")
+    @argument("--authentication", type=str, default=NONE_AUTH, choices=POSSIBLE_AUTH, \
```

Comment:
Can you update the widgets that serve as UI for this too like in https://github.com/jupyter-incubator/sparkmagic/pull/284/files#r83328805